packages/cloud-cognitive/src/components/Datagrid/_storybook-styles.scss

```diff
@@ -33,3 +33,7 @@ $block-class: #{$pkg-prefix}--datagrid;
     border-bottom: none;
   }
 }
+
+.#{$block-class}__hidden--columns.#{$block-class}__datagridWrap-simple {
+  height: calc(100% - 80px);
```

Comment:
Can we clarify what this calc is for? Also we typically use `spacing` or `rem` instead of pixel values.

Reply:
This is only for storybook display. There is information listed under the table about the hidden column. I'll convert the px values into rem.
![Screenshot 2022-06-24 at 3 29 22 PM](https://user-images.githubusercontent.com/305492/175512807-b8b3a221-fbbe-463e-be41-e01bff15740c.png)